### PR TITLE
Added status badge, artifact download instructions

### PR DESCRIPTION
**Pull request by memtech3**
Added status badge and instructions on how to download prebuilt artifacts to readme. This lowers the barrier to testing as it allows those who would generally be afraid of downloading and compiling themselves to quickly download a prebuilt binary and run it without having to touch the terminal.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
 # SysId: System Identification for Robot Mechanisms
+[![Build](https://github.com/wpilibsuite/sysid/actions/workflows/build.yml/badge.svg)](https://github.com/wpilibsuite/sysid/actions/workflows/build.yml)
 
 This is the C++ version of [frc-characterization](https://github.com/wpilibsuite/frc-characterization). It uses the [wpimath](https://github.com/wpilibsuite/allwpilib/tree/main/wpimath) backend for generating feedforward and feedback gains.
 
+## Downloading and Running Development Versions of SysID
```
Comment (prateekma) [inline README.md:6]: Replace occurrences of `SysID` with `SysId` -- this is the proper capitalization.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,11 @@
...
+
+The SysID github repository uses github actions to build and test each commit. To download the latest development version of SysID, click on the Actions tab, select the branch that you want to view results for (main for least buggy), select the build action filter on the right, select the run of your choosing (the top most run is the latest), scroll down to artifacts, and select the appropriate build for your operating system. You can also click [here](https://github.com/wpilibsuite/sysid/actions/workflows/build.yml?query=branch%3Amain) to go directly to the listing of build action results for the main branch.
```
Comment (prateekma) [inline README.md:8]: github actions -> GitHub Actions

**Review by prateekma — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (calcmogul) [inline README.md:8]: Please leave a blank line before headings. Also, remove the spaces at the end of the line.

Comment (calcmogul) [reply README.md:8]: And also random instances of "github" should be "GitHub".

**Comment by Daltz333** [comment]
Please note that the prebuilt binaries aren't especially useful, you need a robot project outputting a compatible format for characterization.

**Comment by calcmogul** [comment]
And the PR for doing that hasn't been merged into SysId yet.

**Comment by memtech3** [comment]
> Please note that the prebuilt binaries aren't especially useful, you need a robot project outputting a compatible format for characterization.

I was told that this version produces much better results than the python version. I was able to successfully feed it an frc-characterization json to convert and the converted file read just fine, once I actually converted it ;)

**Comment by calcmogul** [comment]
> I was told that this version produces much better results than the python version.

For the analysis part, yeah. However, I was using #78 and #65 for the data on Chief Delphi, and we haven't merged #78 into main yet.

File: README.md [hunk 3]
```diff
@@ -1,7 +1,24 @@
...
+## Downloading and Running Development Versions of SysID
+
+The SysID GitHub repository uses GitHub Actions to build and test each commit. 
+
+### To download the latest development version of SysID from the main branch (most stable): 
+
+1. Click [here](https://github.com/wpilibsuite/sysid/actions/workflows/build.yml?query=branch%3Amain)
+2. Select the run of your choosing (the top most run is the latest)
+3. Scroll down to artifacts, and select the appropriate build for your operating system.
+4.
```
Comment (calcmogul) [inline README.md:15]: Remove `4.` and leave the line blank.

File: README.md [hunk 4]
```diff
@@ -1,7 +1,24 @@
...
+
+1. Click [here](https://github.com/wpilibsuite/sysid/actions/workflows/build.yml?query=branch%3Amain)
+2. Select the run of your choosing (the top most run is the latest)
+3. Scroll down to artifacts, and select the appropriate build for your operating system.
+4. 
+### To download the latest development version of SysID from all branches:
+
+1. Click [here](https://github.com/wpilibsuite/sysid/actions/workflows/build.yml)
+2. Select the run of your choosing (the top most run is the latest)
+3. 3. Scroll down to artifacts, and select the appropriate build for your operating system.
```
Comment (calcmogul) [inline README.md:20]:
```suggestion
3. Scroll down to artifacts, and select the appropriate build for your operating system.
```

**Review by calcmogul — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (memtech3) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (memtech3) [reply README.md]: Done

Comment (memtech3) [reply README.md]: Is the spacing still an issue in the latest commit (just committed)?

File: README.md (hunk 3 above)
Comment (memtech3) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (memtech3) [reply README.md]: Done

**Comment by memtech3** [comment]
Why does the formater not like my pr ?

**Comment by Piphi5** [comment]
[Here](https://github.com/wpilibsuite/sysid/pull/92/checks?check_run_id=2327626707) is the output.

Just run wpiformat and it will handle it for you

**Comment by memtech3** [comment]
rebased from upstream main to fix merge conflict

**Comment by Piphi5** [comment]
Can you squash all of the commits?

**Comment by memtech3** [comment]
I think you have to squash it by selecting squash and merge in the merge menu

**Comment by Piphi5** [comment]
You can squash it from your end with an interactive rebase

**Comment by memtech3** [comment]
I am having no luck merging on my end, can you use the git squash and merge feature on your end?

**Review by prateekma — APPROVED** [review]

**Comment by memtech3** [comment]
Look ma! I'm contributing!

**Pull request merged**